CRM/Core/BAO/Address.php

```diff
@@ -495,15 +495,16 @@ public static function &getValues($entityBlock, $microformat = FALSE, $fieldName
       if (!empty($address->state_province_id)) {
         $address->state = CRM_Core_PseudoConstant::stateProvinceAbbreviation($address->state_province_id, FALSE);
         $address->state_name = CRM_Core_PseudoConstant::stateProvince($address->state_province_id, FALSE);
+        $values['state_province_abbreviation'] = $address->state;
```

Comment:
@mlutfy which one is meant to be the abbreviation?

Reply:
@seamuslee001 oops, good catch, auto-complete typo (fixed). I guess that would be a good use-case for a test?